Make fails with: c99 called with non ISO C99 option -std=c11

A plain "make" fails with the following error:

```
$ make
c99 -g -std=c11 -Wall -Wextra -pedantic  -o iocccsize iocccsize.c
c99 called with non ISO C99 option -std=c11
make: *** [Makefile:38: iocccsize] Error 1
```

Tested on Debian 10 (buster) with GCC 8.3.0.

I'm not sure where this "c99" came from. Note that no environment variable are set:

```
$ set | grep c99
$ export | grep c99
$
```

--- a/Makefile
+++ b/Makefile
@@ -11,6 +11,7 @@
 
 .POSIX :
 
+CC= cc
 GIT= git
 RM= rm
 

